python/tk_multi_loader/model_publishhistory.py

```diff
@@ -82,6 +82,7 @@ def load_data(self, sg_data):
         filters = [ ["project", "is", sg_data["project"] ],
                     ["name", "is", sg_data["name"] ],
                     ["entity", "is", sg_data["entity"] ],
+                    ["sg_status_list", "not_in", ["omt", "ip"]],
```

Comment:
That line is already in the code. But when I checked the yml file there were no publish filters to be gotten. The only publish filters that are set are the ones per individual entity type whereas the publishhistory was getting the main common publish_filters which was set to []
I'll remove this line from this PR and just keep the removal of the ip/omt from the drop down menu. I'll create a pr for the tank config to fix this problem.